sdk/mapper.js

```diff
@@ -66,8 +66,12 @@ class Mapper {
         }
       }
     };
-    if (responseApi.hasReturn()) {
-      payload[m.command_reply][m.result][m.response_return] = responseApi.getReturnType();
+    try {
+      if (responseApi.hasReturn()) {
+        payload[m.command_reply][m.result][m.response_return] = responseApi.getReturnType();
```

Comment:
ok, I didn't write that piece of the code, just surrounded it with a try catch. I'll fix that and probably it's related test now.